### DISCUSSION TITLE
基于InternLM的煤矿安全助手

### DISCUSSION THREAD

**Discussion by sunxiaobei** [InternLM/Tutorial#504] [project]
- 基于InternLM实现智能助手
- 基于煤矿安全领域知识进行微调

**Comment by HPUhushicheng** [comment]
还可以加入不😃

**Reply by sunxiaobei** [reply]
不好意思，之前一直在忙其他事情，搁置了

**Comment by HPUhushicheng** [comment]
好的



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年06月01日 22:43 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [InternLM/Tutorial] 基于InternLM的煤矿安全助手 (Discussion #504) |

不好意思，之前一直在忙其他事情，搁置了
